templates/admin/base_site.html

```diff
@@ -1,15 +1,17 @@
 {% extends "admin/base.html" %}
 {% load i18n rev %}
+{% load static %}
```

Comment:
Minor thing but it's a bit neater to load static in the line above & avoid using load twice.

Reply:
Noted. Done

Reply:
Sorry! I meant, `{% load i18n rev static %}` but was reviewing from my phone so didn't include an example. So only load in what's needed in 1 line, rather than static separately.